test/python/scripts/run_ocm.py

```diff
@@ -48,8 +50,8 @@ def run_thru_ocm(path, ov_ver, device):
   for f in files:
     cmd = [ov_ocm_path, f, device, ov_version]
     test_name,ext=os.path.splitext(f[10:].replace("/","_"))
-    if test_name in invalid_list:
-      continue
+    # if test_name in invalid_list:
```

Comment:
Why is this commented out? We should be skipping invalid tests.

Reply:
Yes, I have uncommented it now